.github/workflows/stream-diff.yml

```diff
@@ -25,13 +25,11 @@ jobs:
           # Switch to base
           git checkout "${GITHUB_BASE_REF}"
 
-          # Get the numerically latest fedora-coreos-stream-generator tag
-          generator_tag=$(
-              curl -H 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' https://api.github.com/repos/coreos/fedora-coreos-stream-generator/git/matching-refs/tags/ |
-              jq -r .[].ref |
-              sed s:^refs/tags/:: |
-              sort -V |
-              tail -n 1)
+          # Get the `git describe` from fedora-coreos-stream-generator
+          git clone https://github.com/coreos/fedora-coreos-stream-generator
```

Comment:
Do we need the full history? Can we make this clone shallow?

Reply:
We technically only need it up to the last tag, but there's no easy way to request that in one shot. There's `--shallow-since` but that wants a date, which would require some more querying to derive. Anyway, the repo doesn't seem heavy enough to warrant more complexity.